modules/apps/29-fee/keeper/escrow.go

```diff
@@ -0,0 +1,106 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+// EscrowPacketFee sends the packet fee to the 29-fee module account to hold in escrow
+func (k Keeper) EscrowPacketFee(ctx sdk.Context, refundAcc sdk.AccAddress, identifiedFee *types.IdentifiedPacketFee) error {
+	// check if the refund account exists
+	hasRefundAcc := k.authKeeper.GetAccount(ctx, refundAcc)
+	if hasRefundAcc == nil {
+		return sdkerrors.Wrap(types.ErrRefundAccNotFound, fmt.Sprintf("Account with address: %s not found", refundAcc))
+	}
+
+	coins := identifiedFee.Fee.ReceiveFee
+	coins = coins.Add(identifiedFee.Fee.AckFee...)
+	coins = coins.Add(identifiedFee.Fee.TimeoutFee...)
+
+	if err := k.bankKeeper.SendCoinsFromAccountToModule(
+		ctx, refundAcc, types.ModuleName, coins,
+	); err != nil {
+		return err
+	}
+
+	// Store fee in state for reference later
+	// feeInEscrow/<port-id>/<channel-id>/packet/<sequence-id>/ -> Fee (timeout, ack, onrecv)
```

Comment:
This can be removed imo. If we change the storage path, we will probably forget to remove this. We can indicate the storage path in the `SetFeeInEscrow` godoc if necessary